src/test/common/clara/test_accumulation.cljc

```diff
@@ -69,6 +69,16 @@
   [& args]
   (apply = args))
 
+;; In ClojureScript the type function does not use the :type key in metadata as in Clojure, but we had
```

Comment:
This is a difference I was not aware of. I know I wrote some tests adding "type metadata" before, so thanks for the cleanup!